File: OdontoSysV2/OdontoSys/OdontoSysModel/src/main/java/com/odontosys/users/model/Persona.java
package com.odontosys.users.model;

public abstract class Persona {
    
    protected String contrasenha;
    protected String nombreUsuario;
    protected String correo;
    protected String telefono;
    protected String nombre;
    protected String apellidos;
    protected TipoDocumento tipoDocumento;
    protected String NumeroDocumento;
    protected TipoUsuario tipoUsuario;
    
    public Persona(){    
    }

    public String getContrasenha() {
        return contrasenha;
    }

    public void setContrasenha(String contrasenha) {
        this.contrasenha = contrasenha;
    }

    public String getNombreUsuario() {
        return nombreUsuario;
    }

    public void setNombreUsuario(String nombreUsuario) {
        this.nombreUsuario = nombreUsuario;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public TipoDocumento getTipoDocumento() {
        return tipoDocumento;
    }

    public void setTipoDocumento(TipoDocumento tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    public String getNumeroDocumento() {
        return NumeroDocumento;
    }

    public void setNumeroDocumento(String NumeroDocumento) {
        this.NumeroDocumento = NumeroDocumento;
    }

    public TipoUsuario getTipoUsuario() {
        return tipoUsuario;
    }

    public void setTipoUsuario(TipoUsuario tipoUsuario) {
        this.tipoUsuario = tipoUsuario;
    }

    @Override
    public String toString() {
        return "Persona{" + "contrasenha=" + contrasenha + ", nombreUsuario=" + nombreUsuario + ", correo=" + correo + ", telefono=" + telefono + ", nombre=" + nombre + ", apellidos=" + apellidos + ", tipoDocumento=" + tipoDocumento + ", NumeroDocumento=" + NumeroDocumento + ", tipoUsuario=" + tipoUsuario + '}';
    }
    
}

File: OdontoSysFrontEnd/OdontoSysModel/Users/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OdontoSysModel.Users
{
    public class Persona
    {
        private string contrasenha;
        private string nombreUsuario;
        private string correo;
        private string telefono;
        private string nombre;
        private string apellidos;
        private string dni;
        private TipoUsuario tipoUsuario;

        public Persona()
        {
        }

        public string Contrasenha { get => contrasenha; set => contrasenha = value; }
        public string NombreUsuario { get => nombreUsuario; set => nombreUsuario = value; }
        public string Correo { get => correo; set => correo = value; }
        public string Telefono { get => telefono; set => telefono = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellidos { get => apellidos; set => apellidos = value; }
        public string Dni { get => dni; set => dni = value; }
        public TipoUsuario TipoUsuario { get => tipoUsuario; set => tipoUsuario = value; }

        public override string ToString()
        {
            return $"Persona{{Contrasenha={Contrasenha}, NombreUsuario={NombreUsuario}, Correo={Correo}, Telefono={Telefono}, Nombre={Nombre}, Apellidos={Apellidos}, DNI={Dni}, TipoUsuario={TipoUsuario}}}";
        }
    }
}
